kg/Dockerfile

```diff
@@ -98,11 +98,10 @@ RUN neo4j-admin import --delimiter='TAB' --skip-duplicate-nodes=true --skip-bad-
     --nodes /sw/indicator_health_nodes.tsv \
     --relationships /sw/indicator_health_edges.tsv
 
-
-
-
 ENV DOCKERIZED="TRUE"
 ENV NEO4J_URL="bolt://localhost:7687"
 
+COPY api.py api.py
+COPY client.py client.py
```

Comment:
Aren't these already added by the git clone above?

Reply:
I tested the endpoint with no cloning of the project in the Dockerfile and it returns the same results when we just rely on the copied files. I think Jim makes a good point about the addition of duplicate files.